packages/datadog-plugin-jest/test/jasmine2.spec.js

```diff
@@ -31,16 +33,20 @@ describe('Plugin', () => {
     maxWorkers: '50%'
   }
 
-  withVersions(plugin, ['jest-jasmine2'], (version, moduleName) => {
+  withVersions('jest', ['jest-jasmine2'], (version, moduleName) => {
```

Comment:
The new tests are really slow. Can their execution be combined? For example, instead of having a single `it` per Jest test file, having multiple ones to avoid having to re-run Jest for each one.

Reply:
I'll combine some, but others are a bit more difficult to combine, such as the focused tests (any other test in the suite is ignored) and beforeEach/afterEach errors (it'd make other tests fail)

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/1967/commits/14a3cb9e651fd852af1e7fce073ed71e1f3cf63e